mojoloader.js

```diff
@@ -211,10 +211,12 @@ if (typeof MojoLoader === 'undefined')
 			if (this.isNode()) {
 				var fs=require('fs');
 				catPath = base+"node_module.js";
-				if (fs.existsSync(catPath)) {
+				try {
+					fs.accessSync(catPath);
 					library.exports = this._propogateGlobals(require(catPath), this._root);
-					return;
+				} catch (e) {
```

Comment:
Wouldn't it be nice to add a console.log here and output the actual error? Which we know already what it will be, but at least it might give a dev or user some clues when certain things don't work. Same would apply to the other PR's.